src/circle.py

```diff
@@ -42,7 +42,7 @@ def circumference(self) -> float:
         """
         return 2 * math.pi * self.radius
 
-    def __eq__(self, other) -> bool:
+    def __eq__(self, other: "Circle") -> bool:
```

Comment:
This is not actually true. `other` is `Any`

Reply:
Good catch. I'll have to remove the typehints in the equals as `Any` needs `typing`.